src/obu.rs

```diff
@@ -393,11 +391,11 @@ fn parse_seq_hdr(c: &mut Rav1dContext, gb: &mut GetBits) -> Rav1dResult<Rav1dSeq
     if color_description_present != 0 {
         pri = gb.get_bits(8) as Rav1dColorPrimaries;
         trc = gb.get_bits(8) as Rav1dTransferCharacteristics;
-        mtrx = gb.get_bits(8) as Rav1dMatrixCoefficients;
+        mtrx = Rav1dMatrixCoefficients::from_repr(gb.get_bits(8) as usize).unwrap();
```

Comment:
What values is this panicking on?  It's unfortunate that `from_repr` just returns an `Option` instead of `Result` so we don't see what the value was in the error message.  Is it `Reserved = 3`?  Or something `> 14`?

Reply:
it seems to be just arbitrary values

```
554/792 dav1d:testdata-8 / Syntax_AV1_mainb8ss420_432x240_001_vq_aom_ctest_4.2  FAIL     0.01 s (exit status 101)

--- command ---
22:37:06 /home/folkertdev/rust/rav1d/tests/../target/release/dav1d -q --threads 1 -i /home/folkertdev/rust/rav1d/build/../tests/dav1d-test-data/8-bit/vq_suite/Syntax_AV1_mainb8ss420_432x240_001_vq_aom_ctest_4.2.stripped.ivf --verify 74cd1c55dc974ee2d13bf6b2a38aeecb
--- stderr ---
thread 'main' panicked at src/obu.rs:397:21:
Rav1dMatrixCoefficients::from_repr(74) will return None
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
-------

555/792 dav1d:testdata-8 / Syntax_AV1_mainb8ss420_432x240_002_vq_aom_ctest_4.2  FAIL     0.00 s (exit status 101)

--- command ---
22:37:06 /home/folkertdev/rust/rav1d/tests/../target/release/dav1d -q --threads 1 -i /home/folkertdev/rust/rav1d/build/../tests/dav1d-test-data/8-bit/vq_suite/Syntax_AV1_mainb8ss420_432x240_002_vq_aom_ctest_4.2.stripped.ivf --verify a9249d720bf9a7dcddce9e358e9aa967
--- stderr ---
thread 'main' panicked at src/obu.rs:397:21:
Rav1dMatrixCoefficients::from_repr(71) will return None
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
-------

556/792 dav1d:testdata-8 / Syntax_AV1_mainb8ss420_432x240_003_vq_aom_ctest_4.2  FAIL     0.01 s (exit status 101)

--- command ---
22:37:06 /home/folkertdev/rust/rav1d/tests/../target/release/dav1d -q --threads 1 -i /home/folkertdev/rust/rav1d/build/../tests/dav1d-test-data/8-bit/vq_suite/Syntax_AV1_mainb8ss420_432x240_003_vq_aom_ctest_4.2.stripped.ivf --verify 713a7b52a34dea39a6ff3f1064fe5a2c
--- stderr ---
thread 'main' panicked at src/obu.rs:397:21:
Rav1dMatrixCoefficients::from_repr(244) will return None
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
-------
```

So that suggests that, to mirror the `dav1d` behavior it should be 


```rust
pub struct Rav1dMatrixCoefficients(u8);

impl Rav1dMatrixCoefficients {
    pub const IDENTITY : Self = Self(0);
    // ...
}
```

that should be fine, because no (exhaustive) matches  are ever performed on this "enum" anyway